fake-manage-recalls-api/stubs/__files/get-success-metrics.json

```diff
@@ -0,0 +1,56 @@
+{
+  "notificationsSent": {
+    "7days": {
+      "achieved": 5,
+      "total": 6,
+      "percentage": 83
+    },
+    "overall": {
+      "achieved": 33,
+      "total": 36,
+      "percentage": 92
+    }
+  },
+  "dossiersIssued": {
+    "7days": {
+      "achieved": 5,
+      "total": 6,
+      "percentage": 83
+    },
+    "overall": {
+      "achieved": 32,
+      "total": 36,
+      "percentage": 89
+    }
+  },
+  "recallsCompleted": {
+    "7days": {
+      "achieved": 6,
+      "total": 6,
+      "percentage": 100
+    },
+    "overall": {
+      "achieved": 35,
+      "total": 36,
+      "percentage": 97
+    }
+  },
+  "timingAveragesSec": [
+    {
+      "label": "Last 7 days",
+      "book": 100,
+      "assess": 80,
+      "dossier": 65,
+      "total": 245,
+      "percentOfLargest": 88
+    },
+    {
+      "label": "All recalls",
+      "book": 120,
+      "assess": 85,
+      "dossier": 73,
+      "total": 278,
+      "percentOfLargest": 100
+    }
```

Comment:
rather than having the 'label' can we do the same structure as elsewhere `7days` and `overall` ?